lighthouse-core/test/gather/fetcher-test.js

```diff
@@ -46,37 +46,7 @@ let fetcher;
 beforeEach(() => {
   connectionStub = new Connection();
   driver = new Driver(connectionStub);
-  fetcher = new Fetcher(driver.defaultSession, driver.executionContext);
-  browserMilestone = 92;
```

Comment:
Do we still need to mock `getBrowserVersion` anymore?

Reply:
nope. merges are hard